library/Icingadb/Web/Controller.php

```diff
@@ -89,6 +97,39 @@ public function createPaginationControl(Paginatable $paginatable)
         $paginationControl->setDefaultPageSize($this->getPageSize(null));
         $paginationControl->setAttribute('id', $this->getRequest()->protectId('pagination-control'));
 
+        if (Url::fromRequest()->getParam(ViewModeSwitcher::DEFAULT_VIEW_MODE_PARAM) === 'minimal') {
+            $hasLimitParam = Url::fromRequest()->hasParam(LimitControl::DEFAULT_LIMIT_PARAM);
+
+            if ($paginationControl->getDefaultPageSize() <= LimitControl::DEFAULT_LIMIT && ! $hasLimitParam) {
+                $paginationControl->setDefaultPageSize($paginationControl->getDefaultPageSize() * 2);
+            }
+        }
```

Comment:
Same as above in `createLimitControl`. Why should the page size be doubled just because there's a `view=minimal` parameter in the url?

Reply:
Same.

Reply:
✅